src/mlpack/core/optimizers/rmsprop/rmsprop.hpp

```diff
@@ -2,8 +2,9 @@
  * @file rmsprop.hpp
  * @author Ryan Curtin
  * @author Marcus Edel
+ * @author Vivek Pal
  *
- * RMSprop optimizer. RmsProp is an optimizer that utilizes the magnitude of
+ * RMSprop optimizer. RMSprop is an optimizer that utilizes the magnitude of
```

Comment:
We should probably call it `RMSProp`. The reason I changed it to `RMSprop` was to make it consistent with the rest of the code. Should I go ahead and do the renaming?